base/file.jl

```diff
@@ -35,6 +35,17 @@ import .Base.RefValue
     pwd() -> AbstractString
 
 Get the current working directory.
+
+# Examples
+```julia-repl
+julia> pwd()
+"/home/JuliaUser/.julia/v0.7"
```

Comment:
The `v0.7` folder is not used anymore with Pkg(3). Just change to `"/home/JuliaUser"` maybe?